src/eslint-config-adeira/getCommonConfig.js

```diff
@@ -82,5 +83,15 @@ module.exports = function getCommonConfig(rules /*: EslintConfigRules */) /*: Es
       'eslint-plugin-adeira',
       'eslint-plugin-prettier',
     ],
+
+    overrides: [
+      {
+        files: ['**/__generated__/*.graphql.js'],
+        rules: {
+          // Relay disables generated files with unlimited scope
```

Comment:
You know we have Relay under control and we could just generate the files with correct Eslint disables? 😅

Reply:
I know, I like your way of thinking, at one point I was considering it too except I don’t know if we should do that - we would disable only what doesn’t match our rules, but user might have different config. Therefore I would suggest to have this and let that idea sink in.